lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -4,6 +4,8 @@
         function(_, gettext, BaseCollection, TopicModel) {
             var TopicCollection = BaseCollection.extend({
                 initialize: function(topics, options) {
+                    this.url = options.url;
```

Comment:
@muhammad-ammar can this logic be moved to  the base collection? A lot of collections and models share this behavior. This will require less changes in individual files.